tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -1271,9 +1278,9 @@ void CorProfiler::InitializeProfiler(WCHAR* id, CallTargetDefinition* items, int
                 MethodReplacement(
                     {},
                     MethodReference(targetAssembly, targetType, targetMethod, EmptyWStr, minVersion, maxVersion,
-                        {}, signatureTypes),
+                        {}, signatureTypes, targetMethodArgumentsToLoad),
                     MethodReference(wrapperAssembly, wrapperType, EmptyWStr, calltarget_modification_action, {}, {}, {},
-                                    {})));
+                                    {}, {})));
```

Comment:
Now that I'm checking this, porting this fix to `release/2.0`  will be a rewrite.

Reply:
Yeah...I can do the rewrite for 2.0